client/FxAccountClient.js

```diff
@@ -193,6 +193,8 @@ define([
    *     @param {String} [options.metricsContext.utmMedium] marketing campaign medium
    *     @param {String} [options.metricsContext.utmSource] marketing campaign source
    *     @param {String} [options.metricsContext.utmTerm] marketing campaign search term
+   *   @param {String} [options.unblockCode]
```

Comment:
That's cause the params above it have extra indentation to handle being 2 levels deep in the `options`.